Core/Libraries/Include/Lib/BaseTypeCore.h

```diff
@@ -0,0 +1,145 @@
+/*
+**	Command & Conquer Generals(tm)
```

Comment:
Minor: In my opinion all source files in Core should say Generals Zero Hour, because that name includes both title names.

Reply:
Looks odd to me if Core is a mix of Generals and Generals Zero Hour. It should be consistent.